### DISCUSSION TITLE
api_server运行的uvicorn.run()中workers不能大于1

### DISCUSSION THREAD

**Discussion by markoov** [zai-org/ChatGLM3#1073] [General]
当workers大于1时，显示报错是模型未定义，即便将model设为全局变量也是一样。
所以想请问一下如何实现多进程呢？
还想和大家讨论一下如何在多进程调用一个llm的条件下如何节约显存的问题。

**Comment by We-IOT** [comment]
你可以直接 uvicorn命令行运行 这样可以开启多个wokers

**Reply by markoov** [reply]
已解决，感谢大佬
